core/toolbox.js

```diff
@@ -170,7 +170,8 @@ Blockly.Toolbox.prototype.init = function() {
           Blockly.hideChaff(true);
         }
         Blockly.Touch.clearTouchIdentifier();  // Don't block future drags.
-      });
+      }, /*opt_noPreventDefault*/ false, /*opt_noPreventDefault*/ true);
```

Comment:
I'm looking at pulling this into Blockly.  Why do you have the same argument twice with different values here?

Reply:
My mistake.

Reply:
@guillaumejenkins looks like there's a mistake here, can you take a look.

Reply:
It's a comment copy/paste mistake, the 1st parameter comment should read `/*opt_noCaptureIdentifier*/`, as seen in the function signature of `Blockly.bindEventWithChecks_` above

Reply:
Great, thanks for the clarification. 

I have a few other questions below to make sure I know exactly what I'm pulling in.

In this case you're setting opt_noPreventDefault because you want a click on the toolbox to dismiss any context menus and also potentially start a scroll, correct?

Reply:
Consolidated answers in the other comment